sdk/samples/iot/sample.c

```diff
@@ -0,0 +1,454 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "sample.h"

```

Comment:
can we have a better name than sample.c/sample.h ?

Reply:
I'm not sure what to call these.  Cris doesn't like using the terms utility or helper, and it is more than that.  Perhaps sample_base.h or sample_foundation.h?  I notice there are a lot of sample_pnp...  files now too, and I'd like to differentiate these.  Thoughts?  Thanks.

Reply:
Iot_sample_foundation.h?

Reply:
I dig it.
I'll go with that and we can update later if needed.  Good news is this file isn't mentioned anywhere in documentation... I don't think...